Parsers/ASimAuditEvent/Parsers/ASimAuditEventDataminrPulse.yaml

```diff
@@ -0,0 +1,91 @@
+Parser:
```

Comment:
Your events are alerts and not audit events, and therefore the parser is irrelevant. Please remove it.

Reply:
@oshezaf 
We did not find any direct schema with which we can map our alert data, So we picked up Audit events which had maximum number of matching fields with our data. If you think we can map our alert data with any other ASIM schema then please suggest.

Reply:
While we do consider one, at this point in time there is no schema for alerts and therefore no ASIM parser is called for. You create an alert from your event by creating an analytic rules that maps entities to the (non-ASIM) alert schema. 

In practice, just remove the ASIM parser. You can have a non-ASIM parser that uses the field names, but not as part of the ASIM distribution but rather as part of the solution.

Reply:
@oshezaf OK Thanks

Reply:
@oshezaf @v-prasadboke 
Regarding the ASIM parser we are working on to find the best possible schema to match our dataminr data.

Reply:
@oshezaf We have removed the ASIM parse because of not finding the suitable schema to map Dataminr Alerts.